dashboard/app/controllers/zendesk_session_controller.rb

```diff
@@ -9,15 +9,40 @@ class ZendeskSessionController < ApplicationController
   SECRET = Dashboard::Application.config.zendesk_secret
   SUBDOMAIN = Dashboard::Application.config.zendesk_subdomain
 
+  @error = nil
```

Comment:
I don't think you can initialize an instance variable right in the class definition like this - it may need to go inside a `def initialize` block.  On the other hand, in Ruby it might be totally unnecessary to initialize an instance variable to `nil`, so you might be able to just drop this line.